[ADD] Appraisals: new appraisal doc (#9835)
odoo/documentation: content/applications/hr/appraisals/new_appraisals.rst

Once all the desired changes are complete, click :guilabel:`Save & Close`.

The meeting now appears on the calendar, and the invited parties are informed, via email.

.. image:: new_appraisals/meeting.png
   :align: center
   :alt: The meeting form with all information entered for Ronnie Hart's annual appraisal.

The other way to schedule a meeting is from the individual appraisal form. To do this, navigate to
the :menuselection:`Appraisal application` dashboard, then click on an appraisal card.

Next, click on the :guilabel:`Meeting` smart button, and the calendar loads. Follow the same
directions above to create the meeting.

For more detailed information on how to schedule activities, refer to the :doc:`activities
<../../essentials/activities/>` documentation.

.. note::
   If no meetings are scheduled, the :guilabel:`Meeting` smart button reads :guilabel:`No Meeting`.

Complete an appraisal
=====================

After the appraisal is complete, and both the manager and employee have met to discuss the
appraisal, the appraisal can be marked as done. When completed, click the :guilabel:`Mark as Done`
button on the appraisal form, located in the top-left corner.

> Once the appraisal is marked as done, the :guilabel:`Mark as Done` button disappears, and a

Reviewer: ```suggestion
Once the appraisal is marked as *Done*, the :guilabel:`Mark as Done` button disappears, and a
```

:guilabel:`Reopen` button appears.

> Modifications are **not** possible once the appraisal is marked as done.

Reviewer: this might be more useful in its own admonition. what do you think?

> To make any changes to an appraisal that is marked as done, click the :guilabel:`Reopen` button,

Reviewer: ```suggestion
To make any changes to an appraisal that is marked as *Done*, click the :guilabel:`Reopen` button.
```

> then click the :guilabel:`Confirm` button that appears, then make any modifications needed. Once all

Reviewer: ```suggestion
Then, click the :guilabel:`Confirm` button that appears, and make any modifications needed. Once all
```

==============
New appraisals
==============

To create a new appraisal for an employee, first navigate to the main *Appraisals* dashboard by
opening the :menuselection:`Appraisals` app. The :guilabel:`Appraisals` dashboard is the default
view.

Appraisals dashboard
====================

All appraisals are displayed on the dashboard in a default Kanban view, with a list of groupings on

> the left side of the dashboard, including :guilabel:`Company`, :guilabel:`Department`, and
> :guilabel:`Status`.

Reviewer 2: Match the UI text exactly please :)
```suggestion
the left side of the dashboard, including :guilabel:`COMPANY`, :guilabel:`DEPARTMENT`, and
:guilabel:`STATUS`.
```

Click any grouping option to view appraisals for **only** the chosen selection.

.. note::
   Only groupings with multiple selections appear in the list. For example, if a database only has

>    one company, the :guilabel:`Company` grouping does **not** appear, since there is no other

Reviewer 2: ```suggestion
   one company, the :guilabel:`COMPANY` grouping does **not** appear, since there is no other
```

   company to select.

Each appraisal card displays the following information:

> - :guilabel:`Name`: the employee's name.
> - :guilabel:`Department`: the department the employee is associated with.
> - :guilabel:`Company`: the company the employee works for. This only appears in a multi-company
>   database.
> - :guilabel:`Date`: the date the appraisal was requested, or is scheduled for in the future.
> - :guilabel:`Activities`: any activities that are scheduled for the appraisal, such as
>   :guilabel:`Meetings` or :guilabel:`Phone Calls.` If no activities are scheduled, an activity can
>   be scheduled from the :icon:`fa-clock` :guilabel:`(clock)` icon, located in the bottom-left corner
>   of an appraisal card.
> - :guilabel:`Manager`: the employee's manager, indicated by the profile icon in the bottom-right
>   corner of an appraisal card.
> - :guilabel:`Status banner`: the status of the appraisal. A banner appears if an appraisal is
>   marked as either :guilabel:`Canceled` or :guilabel:`Done`. If no banner is present, that means the
>   appraisal has not happened, or has not been scheduled yet.

Reviewer 2: I agree with KSC here, and suggest the following since there are no field names visible for a guilabel.

Also linking to the activities doc instead of explaining how to schedule an activity from this view, which breaks the flow IMO
```suggestion
- **Name**: the employee's name.
- **Department**: the department the employee is associated with.
- **Company**: the company the employee works for. This only appears in a multi-company database.
- **Date**: the date the appraisal was requested, or is scheduled for in the future.
- **Activities**: any :doc:`activities <../../essentials/activities>` that are scheduled for the
  appraisal, such as *Meetings* or *Phone Calls*.
- **Manager**: the employee's manager, indicated by the profile icon in the bottom-right corner of
  an appraisal card.
- **Status banner**: the status of the appraisal. A banner appears if an appraisal is marked as
  either *Canceled* or *Done*. If no banner is present, that means the appraisal has not happened,
  or has not been scheduled yet.
```

To view the details of any appraisal, click on the card to open the appraisal form.

.. image:: new_appraisals/dashboard.png
   :align: center
   :alt: The Appraisals dashboard with each appraisal in its own box.

Create an appraisal
===================

To create a new appraisal, click the :guilabel:`New` button in the upper-left corner of the
:guilabel:`Appraisals` dashboard. Doing so reveals a blank appraisal form. After entering a name in
the first blank field, proceed to enter the following information on the form:

- :guilabel:`Manager`: select the employee's manager from the drop-down menu. The manager is

>   responsible for completing the :guilabel:`Manager's Feedback` section of the appraisal. This field

Reviewer 2: This section text is not visible in this current UI
```suggestion
  responsible for completing the *Manager's Feedback* section of the appraisal. This field
```

  auto-populates after the employee is selected, if they have a manager set on their employee
  profile.
- :guilabel:`Appraisal Date`: the current date is automatically entered in this field. This field is

>   automatically updated once the appraisal is completed or canceled, with the corresponding date of

Reviewer 2: The dreaded double-L'ed cancelled 😢
```suggestion
  automatically updated once the appraisal is completed or cancelled, with the corresponding date of
```

  completion or cancellation.
- :guilabel:`Department`: select the employee's department from the drop-down menu. This field
  auto-populates after the employee is selected, if they have a department set on their employee
  profile.
- :guilabel:`Company`: select the employee's company from the drop-down menu. This field
  auto-populates after the employee is selected, if they have a company set on their employee
  profile.

.. note::
   The only required fields for the appraisal form are the employee's name, the :guilabel:`Manager`,
   and the :guilabel:`Company`.

Once the form is complete, click the :guilabel:`Confirm` button to confirm the appraisal request.

Once confirmed, the employee receives an email stating that an appraisal was requested, and is then
prompted to schedule an appraisal date.

The status changes to :guilabel:`Confirmed`, and the :guilabel:`Employee's Feedback` section of the
:guilabel:`Appraisal` tab is grayed out. The information in that section only appears after the
self-assessment is published by the employee. The :guilabel:`Final Rating` field also appears once
the appraisal request is confirmed.

> If there are any existing appraisals for the employee, an *Appraisal* smart button appears at the
> top of the page, listing the total number of appraisals there are for the employee.

Reviewer 2: ```suggestion
If there are any existing appraisals for the employee, an :guilabel:`Appraisal` smart button appears
at the top of the page, listing the total number of appraisals there are for the employee.
```

:guilabel:`Skills` tab of the appraisal form auto-populates with the skills from the :ref:`employee
form <employees/skills>`, once an appraisal is confirmed.

Each skill is grouped with like skills, and the :guilabel:`Skill Level`, :guilabel:`Progress`, and
:guilabel:`Justification` are displayed for each skill.

Update any skills, or add any new skills to the :guilabel:`Skills` tab.

If a skill level has increased, a reason for the improved rating can be entered into the
:guilabel:`Justification` field, such as `took a fluency language test` or `received Javascript
certification`.

.. seealso::
   Refer to the :ref:`Create a new employee <employees/skills>` document for detailed instructions
   on adding or updating a skill.

After an appraisal is completed, and the skills have been updated, the next time an appraisal is
confirmed, the updated skills populate the :guilabel:`Skills` tab.

.. image:: new_appraisals/skills.png
   :align: center
   :alt: The skills tab of an appraisal form, all filled out.

.. note::
   The :guilabel:`Skills` tab can be modified **after** the employee and their manager have met and
   discussed the employee's appraisal.

>    This is a common situation as the manager may not have all the necessary information to proerly

Reviewer 2: Typo
```suggestion
   This is a common situation as the manager may not have all the necessary information to properly
```

modifications, such as designating a :guilabel:`Start` time, or modifying the default
:guilabel:`Title` to the meeting.

Add the appraisee in the :guilabel:`Attendees` section, and include anyone else who should also be
in the meeting, if necessary.

To make the meeting a video call, instead of an in-person meeting, click :icon:`fa-plus`
:guilabel:`Odoo meeting`, and a :guilabel:`Videocall URL` link appears in the field.

Once all the desired changes are complete, click :guilabel:`Save & Close`.

The meeting now appears on the calendar, and the invited parties are informed, via email.

.. image:: new_appraisals/meeting.png
   :align: center
   :alt: The meeting form with all information entered for Ronnie Hart's annual appraisal.

The other way to schedule a meeting is from the individual appraisal form. To do this, navigate to
the :menuselection:`Appraisal app` dashboard, then click on an appraisal card.

Next, click on the :icon:`fa-calendar` :guilabel:`Meeting` smart button, and the calendar loads.
Follow the same directions above to create the meeting.

For more detailed information on how to schedule activities, refer to the :doc:`activities

> <../../essentials/activities/>` documentation.

Reviewer 2: Nitpick
```suggestion
<../../essentials/activities>` documentation.
```
